Sources/AppStoreConnectCLI/Services/Operations/InviteTesterOperation.swift

```diff
@@ -78,12 +78,12 @@ struct InviteTesterOperation: APIOperation {
             .data
             .id
 
-        return GetBetaTesterInfoOperation(
-                options: GetBetaTesterInfoOptions(id: testerId)
-            )
-            .execute(
-                with: .init(betaTesterResponse: dependencies.betaTesterResponse)
-            )
+        let endpoint = APIEndpoint.betaTester(
+            withId: testerId,
+            include: [.betaGroups, .apps])
+
+        return dependencies.betaTesterResponse(endpoint)
+            .map { BetaTester($0.data, $0.included) }
```

Comment:
Because the requirement here is to use betatester ID to get info (more accurate), and in our `GetBetaTesterInfoOperation ` I use the email. So `GetBetaTesterInfoOperation ` can't be used here

Reply:
Do you think there is room to break apart the operation for reuse?

Reply:
We can. It will be something like `GetBetaTesterInfoByEmailOperation` and `GetBetaTesterInfoByIdOperation`. Emm... what do you think?